src/ZendeskApi.Client.Tests/Resources/AssignableGroupResourceFixture.cs

```diff
@@ -1,49 +0,0 @@
-﻿using System;
```

Comment:
Im totally working on that... I didnt like the previous tests, as they wernt really validating the responses.... What they were doing was to validate that a long turned to an int and that got passed to a method.

Take a look at the new tests. The approach is to replace the old tests with true in memory end to end tests.